action.yml

```
@@ -18,9 +24,16 @@ outputs:
   terraform-docs-version:
     description: The version of terraform-docs to use.
     value: ${{ steps.terraform-docs.outputs.version }}
+
 runs:
   using: "composite"
   steps:
+    - id: go-junit-report
+      run: echo "version=2.0.0" >> $GITHUB_OUTPUT
+      shell: bash
+    - id: go-mock
+      run: echo "version=1.6.0>> $GITHUB_OUTPUT
```

Add a space for consistency.
```suggestion
      run: echo "version=1.6.0 >> $GITHUB_OUTPUT
```